edxval/tests/test_api.py

```diff
@@ -1055,3 +1055,47 @@ def test_invalid_encoded_video_attr(self):
     def test_invalid_course_id(self):
         xml = self.make_import_xml(video_dict=constants.VIDEO_DICT_FISH)
         self.assert_invalid_import(xml, "x" * 300)
+
+
+class GetCourseVideoRemoveTest(TestCase):
+    """
+    Tests for our remove_video_for_course function in api.py
+    """
+
+    def setUp(self):
+        """
+        Creates Video objects in database
```

Comment:
I would have written it as `Creates video objects for courses` or something like that.